README.md

```diff
@@ -318,7 +315,7 @@ $ port install kubeseal
 If you just want the latest client tool, it can be installed into
 `$GOPATH/bin` with:
 
-```sh
+```bash
```

Comment:
Why bash here?

Of we opt for showing all examples as console interactions then let's use the appropriate prompt here (iirc GitHub requires `$`) and use `console`